src/components/BoardContent/BoardContent.js

```diff
@@ -41,7 +42,25 @@ function BoardContent() {
 	}
 
 	const onColumnDrop = (dropResult) => {
-		console.log(dropResult);
+		let newColumns = [...columns];
+		newColumns = applyDrag(newColumns, dropResult);
+
+		let newBoard = { ...board };
+		newBoard.columnOrder = newColumns.map((c) => c.id);
+		newBoard.columns = newColumns;
+
+		setColumns(newColumns);
+		setBoard(newBoard);
```

Comment:
I wonder, why we need to setBoard here?
And another question about the intialData, why columns and cards are positioned by order, but still need "cardOrder", "columnOrder"